### PR TITLE
[RI-184] : Documentation - Auto-discovery flows for both on-premise Redis Enterprise and Redis Cloud databases

### PR DESCRIPTION
**Pull request by Arushi201296**
**For release** : 1.3.0
**Section of documentation** : Using RedisInsight (` /ri/using-redisinsight/auto-discover-databases/`)
**Description** : This PR adds a new section in the documentation which lists the steps to be followed in order to add on-premise Redis Enterprise and Redis Cloud databases.

### CODE REVIEW

File: content/ri/using-redisinsight/auto-discover-databases.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+- **Step 7**: A page containing all the active databases in the selected subscriptions will be displayed. Choose the databases which you want to register and click on **REGISTER DATABASES**.
+
+![rc-step-7](/images/ri/rc-step-6.png)
+
+- **Step 8**: A page containing successfully added and failed to add databases would be displayed. If all databases were successfully added, you would see a message saying 'All selected databases added successfully'.
+
+![rc-step-8](/images/ri/rc-step-7.png)
+
+- **Step 9**: Click on **View databases** to see the auto discovered RC databases.
```
Comment (slemeur) [inline content/ri/using-redisinsight/auto-discover-databases.md]: RC --> Redis Cloud

File: content/ri/using-redisinsight/auto-discover-databases.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+Title: Auto discover databases
+weight: 10
+categories: ["RI"]
+path: auto-discover-databases/
+---
+This page lists the steps to be followed in order to automatically add on-premise Redis Enterprise and Redis Cloud databases.
```
Comment (slemeur) [inline content/ri/using-redisinsight/auto-discover-databases.md]: Redis Cloud Pro

File: content/ri/using-redisinsight/auto-discover-databases.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Auto discovering on-premise Redis Enterprise databases
+
+Redis Enterprise databases can be automatically discovered and added in RedisInsight.
```
Comment (slemeur) [inline content/ri/using-redisinsight/auto-discover-databases.md]: Redis Enterprise (RE)

File: content/ri/using-redisinsight/auto-discover-databases.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+- **Step 5**: Select the databases which you want to add from the list and click on **ADD SELECTED DATABASES**.
+
+![re-step-5](/images/ri/re-step-5.png)
+
+- **Step 6**: All the databases which were successfully added will be dislayed on the screen. Click on **View databases** in order to see them in the databases page.
+
+## Auto discovering Redis Cloud databases
+
+Redis Cloud databases can also be automatically discovered and added in RedisInsight.
```
Comment (slemeur) [inline content/ri/using-redisinsight/auto-discover-databases.md]: Redis Cloud (RC)

**Comment by slemeur** [comment]
Also please use "Redis Enterprise Cloud"

File: content/ri/using-redisinsight/auto-discover-databases.md (hunk 2 above)
Comment (Arushi201296) [reply content/ri/using-redisinsight/auto-discover-databases.md]: Would it be better if I place Pro in brackets?
Like this `..... and Redis Enterprise Cloud ( Pro ) databases`  
Also, place a note below it like : 
Doc-Pro

**Comment by bmansheim** [comment]
@Arushi201296 Great job writing the procedure! Please review my commit as an example of how to match our standard formatting and style. Some of these changes are described in https://docs.redislabs.com/latest/contribution-guide/.

**Comment by Arushi201296** [comment]
> @Arushi201296 Great job writing the procedure! Please review my commit as an example of how to match our standard formatting and style. Some of these changes are described in https://docs.redislabs.com/latest/contribution-guide/.

I am glad that you liked it @bmansheim . I also appreciate the feedback and I am looking forward to contribute more valuable contents to the documentation.

**Pull request merged**